node/common/client_service_base.ts

```diff
@@ -745,7 +745,7 @@ export class ClientServiceBase {
     if (
       !isString(id) ||
       !isString(name) ||
-      functionBytes.constructor.name !== 'Uint8Array'
+      !(functionBytes instanceof Uint8Array)
```

Comment:
This is a bug.
We should use instanceof to check if the type.
This is because, for example, Node's Buffer instance inherits JavaScript Uint8Array so if we use `functionBytes.constructor.name !== 'Uint8Array'` then the check will fail.